modules/javafx.web/src/main/native/Source/WebCore/platform/network/java/SocketStreamHandleImplJava.cpp

```diff
@@ -103,14 +104,9 @@ void SocketStreamHandleImpl::platformSend(const uint8_t* data, size_t len, Funct
 
     jint res = env->CallIntMethod(m_ref, mid, (jbyteArray) byteArray);
     if (WTF::CheckAndClearException(env)) {
-        completionHandler(false);
-    } else {
-        completionHandler(res == (int)len);
+        return { };
     }
-}
-
-void SocketStreamHandleImpl::platformSendHandshake(const uint8_t*, size_t, const Optional<CookieRequestHeaderFieldProxy>&, Function<void(bool, bool)>&&)
-{
+    return { res };
```

Comment:
This fails to compile on Windows x64 with:

```
modules\javafx.web\src\main\native\Source\WebCore\platform/network/java/SocketStreamHandleImplJava.cpp(109): error C2398:
 Element '1': conversion from 'jint' to 'const T &' requires a narrowing convers
ion
        with
        [
            T=uint64_t
        ]
```

I think you will need a `static_cast<size_t>`